### PR TITLE
Added bounce ball game

### LINKED ISSUE #4218: [New game]: Adding a bouncing ball game
### 🎮 Game Request

Game logic and basic description
This HTML code represents a Bouncing Ball Game that you can play using the left, right, and up arrow keys. The game canvas displays a map created with customizable and scriptable tiles, each having various properties like color, solidity, bounce, friction, gravity, and more.

### Point down the features

Game points
Players can control a ball character on the map using the left, right, and up arrow keys to move and interact with the different tiles. For example, some tiles are solid, and the ball can bounce off them, while others trigger scripts when touched.

Uploading Recording 2024-06-04 214855.mp4…

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by YadlaMani**
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes #4218 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @YadlaMani ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by YadlaMani** [comment]
@kunjgit fixed the readme with the changes you requested

**Comment by YadlaMani** [comment]
@kunjgit  done the requested changes

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @YadlaMani,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Comment by YadlaMani** [comment]
@kunjgit yeah done the changes

File: assets/images/Screenshot 2024-06-04 224119.png
Comment (ishita-43) [inline assets/images/Screenshot 2024-06-04 224119.png:1]: rename the image per naming convention mentioned in the contribution guidelines

File: README.md [hunk 1]
```diff
@@ -324,12 +324,18 @@ This repository also provides one such platforms where contributers come over an
 |[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game)   |
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light)   |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)
```
Comment (ishita-43) [inline README.md:328]: remove the duplicate entry

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Review by ishita-43 — APPROVED** [review]
Ready to merge

**Comment by kunjgit** [comment]
Thank you  @YadlaMani , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**